### ISSUE TITLE
Multiplayer (TCP/IP) Character Progress Not Saving For Non-Host Player

### ISSUE THREAD

**Issue by djgrubic** [ferib/D2R-Offline#65]
I have been able to successfully have other players, both on and off my local network join my hosted game, but if anyone other than the game host "Saves and Exits" the game, their progress does not save.

If I (hosting the game), "Save and Exit" it seems to be saving their progress for them.

Has anyone else encountered this?

**Comment by mfrklic** [comment]
I do not have much experience with LAN D2 play with classic but I am going to assume these issues may have already existed. I have played a couple games with my brother and step dad over Hamachi vlan and I was the host and had a crash which caused some loss. We also noticed that the last waypoint did not save for everyone.

I would hope the clients could do save and exist to force a save but maybe the host has to be the one to do it and if so maybe just make sure to continue to go down that path. I feel like TCP/IP is there but Blizzard focused their multiplayer efforts on their server side vs local LAN support.

Hopefully someone with more experience with class D2 LAN plan remembers if this is normal or strange new case with Alpha.
